File: java/Disk.java
public class Disk {
    //atribut
    private String type;
    private String capacity;
    private int price;

    //konstruktor
    Disk(){ }

    Disk(String type, String capacity, int price){
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    //prosedur Setter data
    public void setType(String type){
        this.type = type;
    }

    public void setCapacity(String capacity){
        this.capacity = capacity;
    }

    public void setPrice(int price){
        this.price = price;
    }

    //prosedur getter data

    public String getType(){
        return this.type;
    }

    public String getCapacity(){
        return this.capacity;
    }

    public int getPrice(){
        return this.price;
    }

}

File: PHP/disk.php
<?php

    class disk
    {
        private $type = "";
        private $capacity = "";
        private $price = 0;

        //konstruktor
        public function __construct($type = "", $capacity = "", $price = 0){
            $this->type = $type;
            $this->capacity = $capacity;
            $this->price = $price;
        }
         #prosedur setter data
        public function setType($type){
            $this->type = $type;
        }

        public function setCapacity($capacity){
            $this->capacity = $capacity;
        }

        public function setPrice($price){
            $this->price = $price;
        }

        #prosedur getter data
        public function getType(){
            return $this->type;
        }

        public function getCapacity(){
            return $this->capacity;
        }

        public function getPrice(){
            return $this->price;
        }

        public function __destruct(){ }
    }
?>